### PR TITLE
Fix example code for spacy-wordnet

### PR DESCRIPTION
**Pull request by polm**
## Description

It looks like in the most recent version of this library, 0.1.0, it's no longer possible to pass the lang parameter to the component separately. Doing so will raise an error.

This change just removes the config argument so the example code works.

### Types of change

docs update

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Review by svlandeg — APPROVED** [review]
Nice catch!

File: website/meta/universe.json [hunk 1]
```diff
@@ -2463,7 +2463,7 @@
                 "# Load an spacy model (supported models are \"es\" and \"en\") ",
                 "nlp = spacy.load('en')",
                 "# Spacy 3.x",
-                "nlp.add_pipe(\"spacy_wordnet\", after='tagger', config={'lang': nlp.lang})",
+                "nlp.add_pipe(\"spacy_wordnet\", after='tagger')",
                 "# Spacy 2.x",
```
Comment (svlandeg) [inline website/meta/universe.json:2467]:
```suggestion
                "# spaCy 2.x",
```

File: website/meta/universe.json [hunk 2]
```diff
@@ -2463,7 +2463,7 @@
                 "# Load an spacy model (supported models are \"es\" and \"en\") ",
```
Comment (svlandeg) [inline website/meta/universe.json:2463]:
```suggestion
                "# Load a spacy model (supported models are \"es\" and \"en\") ",
```

File: website/meta/universe.json [hunk 3]
```diff
@@ -2463,7 +2463,7 @@
...
                 "nlp = spacy.load('en')",
```
Comment (svlandeg) [inline website/meta/universe.json:2464]: We should probably update this, too?

Comment (polm) [reply website/meta/universe.json:2464]: It would be good to update it, but then the code wouldn't work with v2, so I guess we can just leave it for now?

Comment (polm) [reply website/meta/universe.json:2464]: Eh, I guess it's pseudocode anyway, I'll add a note.

**Comment by svlandeg** [comment]
Looks good to merge!

**Pull request merged**